<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Minotaur</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>611</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="log_interface">
     <item>
      <widget class="QLabel" name="log_title">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Log&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="log_viewer">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Log Console&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Noto Sans'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="main_menu_bar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="setup_menu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Setup</string>
    </property>
    <addaction name="separator"/>
    <addaction name="start_python_interpreter"/>
    <addaction name="separator"/>
    <addaction name="open_serial_box"/>
    <addaction name="open_status_box"/>
    <addaction name="open_parameter_box"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Video</string>
    </property>
    <addaction name="open_camera_display"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_invert_x_axis"/>
    <addaction name="action_invert_y_axis"/>
    <addaction name="separator"/>
    <addaction name="action_clear_log"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="open_about"/>
   </widget>
   <addaction name="setup_menu"/>
   <addaction name="menuVideo"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="open_serial_box">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Serial</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="open_camera_display">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open Camera Display</string>
   </property>
  </action>
  <action name="action_invert_x_axis">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Invert X-Axis</string>
   </property>
  </action>
  <action name="action_invert_y_axis">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Invert Y-Axis</string>
   </property>
  </action>
  <action name="action_clear_log">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Clear Log</string>
   </property>
  </action>
  <action name="actionFigure_Eight">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Figure Eight</string>
   </property>
  </action>
  <action name="actionBox_Manoeuvre">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Box Manoeuvre</string>
   </property>
  </action>
  <action name="actionTriangle_Manoeuvre">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Triangle Manoeuvre</string>
   </property>
  </action>
  <action name="actionMove_to_Circle_Start">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Move to Circle Start</string>
   </property>
  </action>
  <action name="actionMove_from_Circle_Start_To_Origin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Move from Circle Start To Origin</string>
   </property>
  </action>
  <action name="actionCircle_Manoeuvre">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Circle Manoeuvre</string>
   </property>
  </action>
  <action name="actionPick_Shapes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pick Shapes</string>
   </property>
  </action>
  <action name="actionNavigate_Maze">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Navigate Maze</string>
   </property>
  </action>
  <action name="actionManual_Keyboard_Input">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Manual Keyboard Input</string>
   </property>
  </action>
  <action name="actionPreview_Image_Recognition">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Preview Image Recognition</string>
   </property>
  </action>
  <action name="actionVideo_Camera">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Video Camera</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="open_about">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="start_python_interpreter">
   <property name="text">
    <string>Python Interpreter</string>
   </property>
  </action>
  <action name="open_parameter_box">
   <property name="text">
    <string>Parameters</string>
   </property>
  </action>
  <action name="open_status_box">
   <property name="text">
    <string>Status Box</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
